qbft/decided.go

```diff
@@ -41,8 +41,6 @@ func (c *Controller) UponDecided(msg *SignedMessage) (*SignedMessage, error) {
 	}
 
 	if isFutureDecided {
-		// sync gap
-		c.GetConfig().GetNetwork().SyncDecidedByRange(types.MessageIDFromBytes(c.Identifier), c.Height, msg.Message.Height)
```

Comment:
did you remove SyncDecidedByRange from the interface?

Reply:
yes, checkout this commit:
https://github.com/bloxapp/ssv-spec/pull/300/commits/23718342351af66ce5c31054bfecf3d981dae4d4